config/routes.rb

```diff
@@ -0,0 +1,22 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+
+  root to: redirect('/api-docs')
+
+  post 'api/v1/register/:username/:name/:email', to: 'users#register'
+  get 'api/v1/login/:username', to: 'users#login'
+
+  get 'api/v1/cars', to: 'cars#index'
+  get 'api/v1/car/:id', to: 'cars#show'
+  post 'api/v1/car', to: 'cars#create'
+  delete 'api/v1/car/:id', to: 'cars#delete'
```

Comment:
- [ ] This bunch of general routes is a great candidate for the #resources Rails method.

A general example would be:

```ruby
  namespace :api do
     namespace :v1 do
      resources :cars, only: [:index, :create, :show, :delete]
    end
  end
```

_This is just an example that might work based on my own experience, please test and modify as needed so it works as intended for your app._

That way, your routes file will be cleaner, and a lot more readable 💪

Reply:
We tried doing this, but it didn't work we also figured we'd write all the routes in the same way instead of writing some using resources and writing others with customized methods 👍

Reply:
[OPTIONAL]Im only making this optional since you said you’ve tried, this is why I’m approving, but you really ought to try again, it can work